server/tests/utils/comment.test.js

```diff
@@ -10,7 +10,7 @@ describe('article notification utility', () => {
 
 describe('articleNotification()', () => {
   const msg = {
-    to: 'laurangift@gmail.com',
+    to: 'laurangift@gmail.co',
```

Comment:
Is there a reason why you're changing from .com to .co? @d-beloved

Reply:
I had to do that because the mail we have there is a real email. So, when the bug for the email notification was fixed, the owner of the mail began to receive many emails each time we are testing the app, so changing the `com` to `co` stopped the barrage of emails being received at that address.